### DISCUSSION TITLE
Problems with my student ID

### DISCUSSION THREAD

**Discussion by carlitoselian** [community/community#53070] [GitHub Education]
I have a problem to upload my student ID. The form only ask me one file (photo), but my ID have my personal information in front of my ID (selfie, my name, my school) and the back of my ID have the expiration date. 

I have tried to send my 2 sides of my ID in one image, but I received an answer that I edited the photo and you don't receive edited photos. 

What can I do? I could upload an small video of my ID with both sides or I could send both sides of my ID in 2 photos, but I don't know how to send them.

Can you help me, please?

**Comment by Sazid99246** [comment]
Same for me 🙃

**Comment by samus-aran** [comment]
Hi @carlitoselian 👋 

We acknowledge that some institutions do not provide dated (same side) Student IDs. In these situations we would recommend you acquire an additional dated piece of evidence. You can take a photo of both next to one another. 

Let us know how you get on 👍
